web/controllers/rules/pr/merge_conflict.ex

```diff
@@ -0,0 +1,65 @@
+defmodule Dwylbot.Rules.PR.MergeConflict do
+  @moduledoc """
+  Check for error when a PR has a merge conflict
+  """
+  alias Dwylbot.Rules.Helpers
+  @github_api Application.get_env(:dwylbot, :github_api)
+
+  def apply?(payload) do
+    payload["action"] == "closed" && payload["pull_request"]["merged"]
+  end
+
+  def check(payload, _get_data?, token) do
+    payload = @github_api.get_data(token, payload, "list_pull_requests")
+
+    actions = payload
+    |> Enum.filter(fn(pr) ->
+      !pr["pull_request"]["mergeable"]
+    end)
+    |> Enum.map(fn(pr) ->
+      [
+        %{
+          remove_assignees: get_assignees_login(pr["issue"]["assignees"]),
+          url: "#{pr["issue"]["url"]}/assignees"
+        },
+        %{
+          comment: error_message(pr["pull_request"]["user"]["login"]),
+          url: pr["pull_request"]["comments_url"]
+        },
+        %{
+          add_assignees: [pr["pull_request"]["user"]["login"]],
+          url: "#{pr["issue"]["url"]}/assignees"
+        },
+        %{
+          replace_labels: ["merge-conflicts"],
+          url: "#{pr["issue"]["url"]}/labels"
+        }
+    ]
+    end)
+    |> Enum.concat
+
+    if length(actions) > 0 do
+      %{
+        error_type: "pr_merge_conflict",
+        actions: actions,
+        wait: Helpers.wait(Mix.env, 10_000, 1000, 1),
```

Comment:
We wait 10s just to let Github update the status of the PRs if there are merge conflict. It's normally quiet quick but this time make sure that we don't miss a merge conflicts